tests/Knp/DoctrineBehaviors/ORM/TranslatableTest.php

```diff
@@ -34,8 +34,8 @@ function()
             {
                 return 'en';
             },
-            'Knp\DoctrineBehaviors\Model\Translatable\Translatable',
-            'Knp\DoctrineBehaviors\Model\Translatable\Translation',
+            array('Knp\DoctrineBehaviors\Model\Translatable\Translatable', 'Knp\DoctrineBehaviors\Model\Translatable\TranslatableProperties'),
+            array('Knp\DoctrineBehaviors\Model\Translatable\Translation','Knp\DoctrineBehaviors\Model\Translatable\TranslationProperties',),
```

Comment:
Maybe it could be good to also have a test in case where a non array value (eg an object) is passed here instead of an array ?

Reply:
Right, and it makes me think that we should provide a compatibility layer to not break BC.